pkgs/by-name/po/poco/package.nix

```diff
@@ -67,8 +80,16 @@ stdenv.mkDerivation rec {
       url = "https://patch-diff.githubusercontent.com/raw/pocoproject/poco/pull/4879.patch";
       hash = "sha256-VFWuRuf0GPYFp43WKI8utl+agP+7a5biLg7m64EMnVo=";
     })
+    # failing on darwin, could perhaps be patched / a fix upstreamed later
+    ./disable-broken-tests.patch
```

Comment:
If the tests only fail on darwin, maybe they should be patched out only darwin.
What do you think?

Reply:
To be honest, I wasn't able to check on anything but darwin yet. The code of most of these failures look like they probably are also a failure on linux though.

Reply:
I haven't seen many conditionally applied patches for tests yet, outside of the largest ports like LibreOffice, and even there only for versions / variants, not for Darwin. Is this something that is usually done in nixpkgs that I haven't yet discovered?

Reply:
From the top of my head I cannot recall on which packages this happened, yet I was asked to apply patches conditionally if they targeted a specific platform.

Reply:
ok. I'll give linux build a try and see if the patches are also required.